packages/lib/gpcircuits/circuits/proto-pod-gpc.circom

```diff
@@ -216,33 +216,37 @@ template ProtoPODGPC (
 
     
     /*
-     * 1 BoundsCheckModule with its inputs & outputs
+     * (MAX_BOUNDS_CHECKS) BoundsCheckModule with its inputs & outputs
      */
     
     // Array of indices of entries whose values will be
     // bounds-checked. This should be padded with -1 if necessary,
     // which refers to the value 0.
     signal input boundsCheckEntryIndices[MAX_BOUNDS_CHECKS];
 
-    // Array of bounds, where the ith element represents the closed
-    // interval that the value of the entry with index
-    // `comparisonEntryIndices[i]` should lie in. This should be
-    // padded with [0, (1 << 64) - 1] if necessary.
-    signal input boundsCheckBounds[MAX_BOUNDS_CHECKS][2];
+    // Arrays of (inclusive) 64-bit unsigned integer bounds, where the
+    // ith element represents the closed interval that the value of
```

Comment:
... ith element of each array represents the min and max bounds that the value of ith entry must lie between